x-pack/plugins/observability/public/pages/slo_edit/components/slo_edit_form_objectives.tsx

```diff
@@ -0,0 +1,123 @@
+  EuiFlexGrid,
+  EuiFlexItem,
+  EuiFormLabel,
+  EuiSelect,
+  EuiSpacer,
+  useGeneratedHtmlId,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { Control, Controller, UseFormWatch } from 'react-hook-form';
+import type { BudgetingMethod, CreateSLOParams } from '@kbn/slo-schema';
+
+import { SloEditFormObjectivesTimeslices } from './slo_edit_form_objectives_timeslices';
+
+export const BUDGETING_METHOD_OPTIONS: Array<{ value: BudgetingMethod; text: string }> = [
+  { value: 'occurrences', text: 'Occurences' },
+  { value: 'timeslices', text: 'Timeslices' },
+];
+
+export const TIMEWINDOW_OPTIONS = [30, 7].map((number) => ({
+  value: `${number}d`,
+  text: i18n.translate('xpack.observability.slos.sloEdit.objectives.days', {
+    defaultMessage: '{number} days',
+    values: { number },
+  }),
+}));
+
+export interface SloEditFormObjectivesProps {
+  control: Control<CreateSLOParams>;
+  watch: UseFormWatch<CreateSLOParams>;
+}
+
+export function SloEditFormObjectives({ control, watch }: SloEditFormObjectivesProps) {
+  const budgetingSelect = useGeneratedHtmlId({ prefix: 'budgetingSelect' });
+  const timeWindowSelect = useGeneratedHtmlId({ prefix: 'timeWindowSelect' });
+
+  return (
+    <>
+      <EuiFlexGrid columns={3}>
+        <EuiFlexItem>
+          <EuiFormLabel>
+            {i18n.translate('xpack.observability.slos.sloEdit.objectives.budgetingMethod', {
+              defaultMessage: 'Budgeting method',
+            })}
+          </EuiFormLabel>
+
+          <Controller
+            name="budgetingMethod"
+            shouldUnregister
+            control={control}
+            rules={{ required: true }}
+            render={({ field }) => (
+              <EuiSelect id={budgetingSelect} options={BUDGETING_METHOD_OPTIONS} {...field} />
+            )}
+          />
+        </EuiFlexItem>
+
+        <EuiFlexItem>
+          <EuiFormLabel>
+            {i18n.translate('xpack.observability.slos.sloEdit.objectives.timeWindow', {
+              defaultMessage: 'Time window',
+            })}
+          </EuiFormLabel>
+
+          <Controller
+            name="timeWindow.duration"
+            shouldUnregister
+            control={control}
+            rules={{ required: true }}
+            render={({ field }) => (
+              <EuiSelect
+                id={timeWindowSelect}
+                options={TIMEWINDOW_OPTIONS}
+                {...field}
+                value={String(field.value)}
+              />
+            )}
+          />
+        </EuiFlexItem>
+
+        <EuiFlexItem>
+          <EuiFormLabel>
+            {i18n.translate('xpack.observability.slos.sloEdit.objectives.targetSlo', {
+              defaultMessage: 'Target / SLO (%)',
+            })}
+          </EuiFormLabel>
+
+          <Controller
+            name="objective.target"
+            shouldUnregister
+            control={control}
+            rules={{ required: true, min: 0.0001, max: 100, validate: (value) => value > 0 }}
```

Comment:
Since the field is in %, the min value should be `0.001` and the max value `99.999`. 
Validation can be `value > 0 & value < 100`